worlds/ahit/test/TestActs.py

```diff
@@ -0,0 +1,35 @@
+from worlds.ahit.Regions import act_chapters
+from worlds.ahit.Rules import act_connections
+from . import HatInTimeTestBase
+
+
+class TestActs(HatInTimeTestBase):
+    def run_default_tests(self) -> bool:
+        return False
```

Comment:
the base function is a property so this override doesn't actually work. the module should also be `snake_case` to be picked up by unittest correctly.
```suggestion
    run_default_tests = False
```